crt_portal/cts_forms/views.py

```diff
@@ -1068,27 +1068,114 @@ def post(self, request):
 
 class SavedSearchView(LoginRequiredMixin, FormView):
 
+    def get_section_args(self, section_filters):
+        section_args = ''
+        for section_filter in section_filters:
+            logging.info(section_filter)
+            section_args += f'&section_filter={section_filter}'
+        return section_args
+
     def get(self, request):
         saved_searches = SavedSearch.objects.filter().all()
-        selected_section = request.GET.get('section', None)
-        if selected_section:
-            saved_searches = SavedSearch.objects.filter(section=selected_section)
+        section_filter = request.GET.getlist('section_filter', [])
+        section_args = self.get_section_args(section_filter)
+        if section_filter:
+            saved_searches = SavedSearch.objects.filter(section__in=section_filter)
         output = {
+            'section_filter': section_args,
             'saved_searches': saved_searches,
+            'form': SavedSearchFilter(request.GET),
         }
         return render(request, 'forms/complaint_view/saved_searches/index.html', output)
 
     def post(self, request):
         saved_searches = SavedSearch.objects.filter().all()
-        selected_section = request.GET.get('section', None)
-        if selected_section:
-            saved_searches = SavedSearch.objects.filter(section=selected_section)
+        section_filter = request.GET.getlist('section_filter', [])
+        section_args = self.get_section_args(section_filter)
+        if section_filter:
+            saved_searches = SavedSearch.objects.filter(section__in=section_filter)
         output = {
+            'section_filter': section_args,
             'saved_searches': saved_searches,
+            'form': SavedSearchFilter(request.GET),
         }
         return render(request, 'forms/complaint_view/saved_searches/index.html', output)
 
 
+class SavedSearchActionView(LoginRequiredMixin, View):
+    form = SavedSearchActions
+
+    def get_section_args(self, section_filters):
+        section_args = ''
+        for section_filter in section_filters:
+            section_args += f'&section_filter={section_filter}'
+        return section_args
+
+    def get(self, request, id=None):
+        """
+        Get saved search to edit
+        """
+        if id:
+            saved_search = get_object_or_404(SavedSearch, pk=id)
+        else:
+            saved_search = SavedSearch()
+        section_filter = request.GET.getlist('section_filter', [])
+        saved_search_form = SavedSearchActions(instance=saved_search)
+        output = {
+            'form': saved_search_form,
+            'section_filter': section_filter,
+        }
+        if id:
+            return render(request, 'forms/complaint_view/saved_searches/actions/update.html', output)
+        else:
+            return render(request, 'forms/complaint_view/saved_searches/actions/new.html', output)
```

Comment:
The "else" is unnecessary here because you returned from the if above

(reduces nesting)

```suggestion
        return render(request, 'forms/complaint_view/saved_searches/actions/new.html', output)
```